pkg/uzip/uzip_test.go

```diff
@@ -58,32 +63,102 @@ func TestFromZip(t *testing.T) {
 		t.Fatalf(`os.ReadFile("testdata/testFolder/subFolder/file4") = _, %v, want nil`, err)
 	}
 
-	if _, err := os.Stat(f1); err != nil {
-		t.Errorf(`os.Stat(%q) =_, %v, want nil`, f1, err)
+	var x []byte
+
+	x, err = ioutil.ReadFile(f1)
+	if err != nil {
+		t.Errorf("open file: %q failed with: %q", f1, err)
+	}
+	if !bytes.Equal(x, f1Expected) {
```

Comment:
bytes.Equal(etc. etc. )
How long is this file? Is it practical to print out the two values? If not, could you print the first few bytes or something?

Reply:
Guess something slipped here. My bad.

Reply:
Wait. I dont understand the issue here. The error prints the two file names. Do you want to see the beginning of the files being printed to console?

Reply:
it's a suggestion. The reason is that sometimes you get an error like this and the files are long gone. Occasionally seeing just the first bit can be helpful. I'll leave it up to you.